people.json

```diff
@@ -126,41 +126,30 @@
             "Virtual reality",
             "Augmented reality",
             "Mixed reality",
-            "HoloLens",
-            "HTC Vive",
-            "Oculus Rift",
-            "Oculus Quest",
-            "LEAP Motion",
-            "VRTK",
-            "Unity (game engine)",
             "Web development",
             "Web scraping",
             "Scripting",
             "DevOps",
-            "Django",
-            "Wagtail",
+            "Django (web framework)",
+            "Wagtail (CMS)",
             "Flask (web framework)",
-            "Mobile app",
-            "Ionic",
-            "Cordova",
-            "Angular JS",
-            "Android (operating system)",
-            "iOS",
             "Python (programming language)",
             "TypeScript",
-            "R (programming language)",
             "Markdown",
             "JavaScript",
-            "C# (programming language)",
-            "Data science",
-            "Data visualization",
-            "Plotly",
-            "Interaction design",
             "Linux",
             "Microsoft Windows",
             "macOS",
             "Version Control",
-            "Git"
+            "Git",
+            "GitHub Projects",
```

Comment:
I don't think this is a Wikipedia main page heading so won't work

Reply:
This has been resolved in latest commit